CanvasColorSpaceProposal.md

```diff
@@ -151,26 +150,14 @@ Canvas contents are composited in accordance with the canvas element's style (e.
 The chromiumance of color values outside of [0, 1] is not to be clamped, and extended values may be used to display colors outside of the gamut defined by the canvas' color space's primaries.
 This is in contrast with luminance, which is to be clamped to the maximum standard dynamic range luminance, unless high dynamic range is explicitly enabled for the canvas element.
 
-### ImageBitmap
```

Comment:
Per comment above, these changes can't be made in the WebGL or WebGPU specs; they need to be made in the WHATWG HTML spec. I think it's worth just leaving this here so that we know we have to make these changes in the WHATWG later.